recipe/meta.yaml

```diff
@@ -65,18 +67,19 @@ requirements:
         - six
         - ply
         - matplotlib-base
+        - yaml-cpp
     run:
-        - pyside2
+        - qt6-main
+        - pyside6
         - vtk
         - {{ pin_compatible('occt', max_pin='x.x.x') }}
-        - qt
+        - {{ pin_compatible('coin3d', max_pin='x.x') }}
+        - {{ pin_compatible('smesh', max_pin='x.x.x') }}
```

Comment:
Why do u need to repin these? Do you use the privates API? Let the packages do their own exporting

Reply:
occt had API-changes for patch-version updates. For the others there are no pinnings in conda-forge-pinning.

Reply:
The solving problems you are experiencing generally get resolved when:

1. The appropriate pinnings are added to each feedstock. 
2. Migrations are stored for the respective packages. 

If you don't use this your users will have trouble installing your package.

Reply:
what do you mean by: "appropriate pinnings are added to each feedstock"?
Are the pinnings not stored in [conda-forge-pinning-feedstock](https://github.com/conda-forge/conda-forge-pinning-feedstock)?

Reply:
You should feel encouraged to make PRs to:

* [smesh](https://github.com/conda-forge/smesh-feedstock/blob/main/recipe/meta.yaml#L18)
* [coin3d](https://github.com/conda-forge/coin3d-feedstock/blob/main/recipe/meta.yaml#L20)

to add the run_exports, then we can start a migration to "launch the newly added run_export feature" like I did:
https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/6055 for an other package.

Reply:
thanks, done in https://github.com/conda-forge/coin3d-feedstock/pull/28 and https://github.com/conda-forge/smesh-feedstock/pull/87